plugin.video.nos/addon.xml

```diff
@@ -11,7 +11,9 @@
     <provides>video</provides>
   </extension>
   <extension point="xbmc.addon.metadata">
-    <news>v4.0.1 (2018-08-11)
+    <news>v4.0.2 (2019-07-25)
```

Comment:
This is only for the last version

Reply:
In the next version please use the news tag only for the changes in the current version